CenterForOpenScience/osf.io
# [ENG-2336] Send all registration metadata to Datacite

## Purpose

Send more and more accurate info to datcite.

## Changes

- changes `ResourceType` to Registration
- add affiliated institutions for creators

## QA Notes

Please make verification statements inspired by your code and what your code touches.
- Verify that new info is getting push to doi.test.datacite.org

What are the areas of risk?

Any concerns/considerations/questions that development raised?

## Documentation

No new docs.

## Side Effects

None that I know of.

## Ticket

https://openscience.atlassian.net/browse/ENG-2336
